src/main/java/com/minecolonies/coremod/colony/requestsystem/resolvers/StandardRetryingRequestResolver.java

```diff
@@ -70,12 +70,12 @@ public int getCurrentReassignmentAttempt()
 
     @Nullable
     @Override
-    public IToken getCurrentlyBeingReassignedRequest()
+    public IToken<?> getCurrentlyBeingReassignedRequest()
     {
         return current;
     }
 
-    public StandardRetryingRequestResolver(final IToken id, final ILocation location)
+    public StandardRetryingRequestResolver(final IToken<?> id, final ILocation location)
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this constructor to comply with Java Code Conventions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orionminecraft.com/coding_rules#rule_key=squid%3AS1213)